html-forms/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="styles.css" href="styles.css">
+    <title>HTML-Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form method="post">
+      <div id="mail"><label for="email">Email</label></div>
+      <input type="email" id="email" name="user_email" placeholder="example@email.com"/>
+
+      <div id="int"><label for="interest">Why is coding cool?</label></div>
+      <select name="code-is" id="interest">
+        <option value="choice0">Select an option</option>
+        <option value="choice1">Because it's interesting</option>
+        <option value="choice2">Because it is fun</option>
+        <option value="choice3">Because it is powerful</option>
+      </select>
+
+      <div id="f-l"><label for="fav">Favorite language (choose one)</label></div>
+      <label for="html"><input type="radio" id="html" name="lang"/>HTML</label>
+      <label for="css"><input type="radio" id="css" name="lang" />CSS</label>
+      <label for="js"><input type="radio" id="js" name="lang"/>JavaScript</label>
```

Comment:
Sorry I missed this one, these radio `input`s should also have a `value` attribute. 
The `value` is what gets passed on during the form submission.